fmriprep/interfaces/reports.py

```diff
@@ -71,7 +71,7 @@ class SubjectSummaryInputSpec(BaseInterfaceInputSpec):
     t2w = InputMultiPath(File(exists=True), desc='T2w structural images')
     subjects_dir = Directory(desc='FreeSurfer subjects directory')
     subject_id = Str(desc='Subject ID')
-    bold = InputMultiPath(File(exists=True), desc='BOLD functional series')
+    bold = traits.List(desc='BOLD functional series')
```

Comment:
Why is this change necessary?

Reply:
Because the `bold_list` it's being passed is a list that can contain sub-lists, requiring `File(exists=True)` was not working for the sub-lists. Unless you can think of another way to check this!